rig.js

```diff
@@ -127,6 +127,30 @@ class RigManager {
     this.localRig.textMesh.sync();
   }
 
+  setLocalAvatarImage(avatarImage) {
+    const geometry = new THREE.CircleBufferGeometry(0.1, 32);
+    const img = new Image();
+    img.src = avatarImage;
+    img.crossOrigin = 'Anonymous';
+    img.onload = () => {
+      texture.needsUpdate = true;
+    };
+    img.onerror = err => {
+      console.warn(err.stack);
+    };
+    const texture = new THREE.Texture(img);
+    const material = new THREE.MeshBasicMaterial({
+      map: texture,
+      side: THREE.DoubleSide,
+    });
+    const avatarMesh = new THREE.Mesh(geometry, material);
+    avatarMesh.position.x = -0.5;
+    avatarMesh.position.y = -0.02;
+    this.localRig.textMesh.add(avatarMesh);
+
+    this.localRig.textMesh.sync();
```

Comment:
If we aren't changing the text mesh, there is no need to sync.

Reply:
1a301bc